snews.php

```diff
@@ -226,7 +226,7 @@ function readAddons() {
 readAddons();
 
 // LANGUAGE VARIABLES
-@include('lang/'.s('language');
+@include('lang/'.s('language').'.php');
```

Comment:
It is not recommended to check if file exists before `include` (or `require`) because basically, `include` (or `require`) will check if file exist (and throw an exception otherwise)